### PR TITLE
OC-14426: Change path where we run git describe --tags

### PR DESCRIPTION
**Pull request by slclinica**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,5 @@ test/client/forms/forms.js
 test-coverage
 coverage.shield.badge.md
 _drafts
+.idea
+enketo-express-oc.iml
```
Comment (MartijnR) [inline .gitignore:29]: Use a global .gitignore instead (on your system). Can't include all editors.

File: app/models/config-model.js [hunk 2]
```diff
@@ -259,7 +259,8 @@ function getThemesSupported( themeList ) {
 }
 
 try {
-    config[ 'version' ] = execSync( 'git describe --tags', { encoding: 'utf-8' } ).trim();
+    // need to be in the correct directory to run git describe --tags
+    config[ 'version' ] = execSync( 'cd ' + config[ 'home directory' ] + '; git describe --tags', { encoding: 'utf-8' } ).trim();
```
Comment (MartijnR) [inline app/models/config-model.js:263]: There must be a better way to do this, no? The script should know it's own location, right? If so, please send PR to enketo/enketo-express for this change.

File: config/default-config.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
     "default theme": "oc",
     "themes supported": [],
     "base path": "",
+    "home directory": "/usr/local/git/enketo-express-oc",
```
Comment (MartijnR) [inline config/default-config.json:30]: seems silly to add its own location as a config item

**Review by MartijnR — CHANGES_REQUESTED** [review]

File: app/models/config-model.js (hunk 2 above)
Comment (MartijnR) [reply app/models/config-model.js:263]: Also use template literal https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: config/default-config.json (hunk 3 above)
Comment (slclinica) [reply config/default-config.json:30]: could you suggest a better way to do this? I can't think of anything.

File: app/models/config-model.js (hunk 2 above)
Comment (slclinica) [reply app/models/config-model.js:263]: the script does not find out its own location, thats why we have to add this. Will use template literal.

File: config/default-config.json (hunk 3 above)
Comment (MartijnR) [reply config/default-config.json:30]: Unfortunately, I don't quite know how to reproduce the problem, so I cannot test myself. But I was thinking of these (possibly with steps down and up using [path.join()](https://nodejs.org/docs/latest-v12.x/api/path.html#path_path_join_paths)):

* `process.cwd()`
* `__dirname`

https://stackoverflow.com/questions/9874382/whats-the-difference-between-process-cwd-vs-dirname

**Comment by MartijnR** [comment]
@slclinica let me know when you'd like me to review again

File: config/default-config.json (hunk 3 above)
Comment (slclinica) [reply config/default-config.json]: I tested it and the __dirname works. Please code review new code, thank you!

Comment (MartijnR) [reply config/default-config.json]: That's great. Nice and simple. I've also added it to enketo/enketo-express.

**Pull request merged**